kernel/src/main/java/org/kframework/parser/inner/ParseInModule.java

```diff
@@ -159,6 +162,26 @@ public void initialize() {
         }
     }
 
+    /**
+     * Print the list of tokens matched by the scanner, the location and the Regex Terminal
+     */
+    public String tokenizeString(String input, Source source) {
+        StringBuilder sb = new StringBuilder("`Match`    (location), Regex Terminal\n");
```

Comment:
it doesn't look like this table will line up in the first column either if the token matched is longer than 10 characters.

Reply:
True. If the token is longer than 10 chars, it's not going to align.
This is a best-effort display. Tokens can be hundreds of chars long and can contain new line.
Not worth trying to align everything.